### ISSUE TITLE
`site.siteMetaData` is not being queried

### ISSUE THREAD

**Issue by nicgordon** [contentful-userland/gatsby-contentful-starter#37]
The starter has the line `const siteTitle = get(this, 'props.data.site.siteMetadata.title');` which ultimately resolves to undefined because the site metadata is not being queried in the graphql query.

Need to add this to make it work:
```
    site {
      siteMetadata {
        title
      } 
    }
```

**Comment by jdahdah** [comment]
I'm just starting out with Gatsby and GraphQL. Would you mind telling me where exactly I need to add this?

**Comment by Putnam14** [comment]
Pull request #41 resolves this issue. The solution is defined in https://www.gatsbyjs.org/tutorial/part-four/#data-in-gatsby

**Comment by stefanjudis** [comment]
Resolved with #41
